src/sass/common.scss

```diff
@@ -169,9 +169,9 @@ fieldset .error-msg {
     margin-top: 3px;
 }
 
-#content {
-    margin-bottom: 20px;
-}
+//#content {
```

Comment:
yes it will unfortunately. i was thinking to fix the other templates too @negarn @binary-hamedanchi